.github/workflows/pkgdown.yml

```diff
@@ -0,0 +1,35 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches:
+      - develop
+      - kuba.update-documentation
```

Comment:
Should CI trigger for this branch?

Reply:
It shouldn't, I forgot to push a commit that was removing that...